task-maker-format/src/ioi/format/italian_yaml/cases_gen.rs

```diff
@@ -521,17 +521,18 @@ where
                 self.subtask_id, score
             )
         })?;
-        let description = if line.len() >= 2 {
-            Some(line[1].as_str().to_string())
+        let name = if line.len() >= 2 {
+            // Remove whitespaces for retrocompatibility with descriptions
```

Comment:
Should we generate a warning if the name is changed by this, to mark that cases.gen should be updated not to have whitespaces in the name?

Reply:
It's a good idea but I don't know how to do it.